exercises/clock/tests/clock.rs

```diff
@@ -328,3 +328,9 @@ fn test_compare_clocks_with_negative_hours_and_minutes() {
 fn test_compare_clocks_with_negative_hours_and_minutes_that_wrap() {
     assert_eq!(Clock::new(18, 7), Clock::new(-54, -11513))
 }
+
+#[test]
+#[ignore]
+fn test_compare_full_day_and_no_day() {
+    assert_eq!(Clock::new(24, 00), Clock::new(0, 0))
```

Comment:
I think for consistency, it would be good if the `00` in this line were to change to `0`, OR the `0` in this line change to `00`. Right?

Reply:
Indeed, thank you! :)